README.md

```diff
@@ -269,14 +269,86 @@ export $(grep -v '^#' .env | xargs)
 ./operator start
 ```
 
-## Misc commnads
+## Remote signer
+
+You may not want the operator service to have direct access to the validator
+keys. Validator keystores do not need to be present directly in the operator.
+The operator can query a remote signer to get signatures for validator
+exit messages. Because the validator exit signatures are split up and
+shared among oracles, the validator exit message needs to be signed by
+specific shares of the validator private key.
+
+These key shares therefore need to be present in your remote signer.
+
+### Remote signer setup
+
+This command will split up the private keys in the keystores directory
+into private key shares. The resulting private key shares are
+then imported to the remote signer. Local keystores are removed
+as a result of this command since they no longer need to be present.
+
+Notes:
+- You will need to run this command every time the oracle set
+  changes. In order to regenerate key shares, make sure to
```

Comment:
> You will need to run this command every time the oracle set
  changes.

Also in a case when `exit_signature_recover_threshold` changes

Reply:
Adjusted the README, is it okay like this or does it deserve a more detailed explanation?

Reply:
It is okay.